custom/icds/tasks/sms.py

```diff
@@ -101,6 +102,7 @@ def send_custom_sms_report(start_date, end_date, email):
                                     email_from=settings.DEFAULT_FROM_EMAIL)
         report_tracker.remove_report_from_list(str(start_date), str(end_date))
         raise e
+    report_tracker.remove_report_from_list(start_date, end_date)
```

Comment:
i think once you realign things in try block, one or more of these calls will become redundant

Reply:
Yep.